R/scalevariableset.R

```diff
@@ -0,0 +1,95 @@
+#' Scale a Displayr Variable Set
+#'
+#' Given a Displayr Variable Set or Q Question, performs a specified
+#' scaling on the data; first converting any categorical variables to
+#' numeric as necessary.
+#' @param data A Displayr Variable Set or Q Question: A numeric or
+#'     factor vector, or a data.frame containing either all numeric
+#'     columns or all factor columns.
+#' @param type Character string specifying the type of scaling to
+#'     perform; one of \code{"standardize"} (the default),
+#'     \code{"center"}, or \code{"unit"}.
+#' @param within.case Logical; should the scaling be performed with
+#'     case (rows) or variables (columns). An error is through if
+#'     \code{TRUE} and \code{data} contains only one variable.
+#' @return A numeric vector if \code{data} contains only a single
+#'     variable or a data.frame containing the scaled values.
+#' @seealso \code{\link{scale}}
+#' @export
+ScaleVariableSet <- function(
```

Comment:
possible type with a new line here?

Reply:
I believe it is mildly more readable to have each parameter on a new line each indented an identical amount.